MSAL/test/integration/native_auth/end_to_end/MSALNativeAuthEndToEndBaseTestCase.swift

```diff
@@ -26,54 +26,65 @@ import XCTest
 import MSAL
 
 class MSALNativeAuthEndToEndBaseTestCase: XCTestCase {
-    let mockAPIHandler = MockAPIHandler()
-    let correlationId = UUID()
-    var defaultTimeout: TimeInterval = 5
-
-    var sut: MSALNativeAuthPublicClientApplication!
-    var usingMockAPI = false
-
-    class Configuration: NSObject {
-        static let clientId = ProcessInfo.processInfo.environment["clientId"] ?? "<clientId not set>"
-        static let authorityURLString = ProcessInfo.processInfo.environment["authorityURL"] ?? "<authorityURL not set>"
-    }
-
-    func mockResponse(_ response: MockAPIResponse, endpoint: MockAPIEndpoint) async throws {
-        try await mockAPIHandler.addResponse(
-            endpoint: endpoint,
-            correlationId: correlationId,
-            responses: [response]
-        )
+    class Constants {
+        static let nativeAuthKey = "native_auth"
+        static let clientIdEmailPasswordKey = "email_password_client_id"
+        static let clientIdEmailCodeKey = "email_code_client_id"
+        static let clientIdEmailPasswordAttributesKey = "email_password_attributes_client_id"
+        static let clientIdEmailCodeAttributesKey = "email_code_attributes_client_id"
+        static let tenantSubdomainKey = "tenant_subdomain"
```

Comment:
We need to make sure to remove this domain and others that we might use after from the Kusto query on the OCE dashboard

Reply:
I think the lab tenant is already removed from our and eSTS dashboard.